usecases/backup/scheduler.go

```diff
@@ -215,6 +215,27 @@ func (s *Scheduler) RestorationStatus(ctx context.Context, principal *models.Pri
 	return st, nil
 }
 
+func (s *Scheduler) Cancel(ctx context.Context, principal *models.Principal, backend, backupID string,
+) error {
+	defer func(begin time.Time) {
+		var err error
+		logOperation(s.logger, "cancel_backup", backupID, backend, begin, err)
+	}(time.Now())
+
+	path := fmt.Sprintf("backups/%s/%s", backend, backupID)
```

Comment:
backupID is not sanitized right ? Maybe with the right input we could have a vulnerability here when building the string.

Do you think it' a concern or something we'll tackle later ?

Reply:
yeah i agree, will take care of it when have the real cancelation  impl

Reply:
however i think it's just the url safety but it won't be a security concern

Reply:
will take care of it in a follow up PR